doc/source/tools/pointwise/pointwise.rst

```diff
@@ -0,0 +1,335 @@
+    - A line was not split when the geometry was separated into subsections. 
+
+
+----
+
+Assigning the boundary conditions and exporting the mesh
+---------------------------------------------------------
+
+- In the **CAE** option, select **Set Dimension** and choose 2D.
+
+- Go to the **CAE** option and choose **Select Solver**. In the list on the left of the screen, select **GMSH**.
+
+- In the **CAE** option, select **Set Boundary Conditions**. Click **New**. Change the **Name** of the new box by double-clicking on it and insert **wall**. Select all the walls in the geometry (hold the ``ctrl`` button on your keyboard as you click) as is presented in the following image and click on the little check box at the left of the name box that was just changed. All the walls should be moved to that set of boundary conditions. The number 6 indicates that 6 boundaries were just moved to that set of boundary conditions. 
+
+.. image:: images/boundary_conditions.png
+    :align: center
+
+- Repeat the previous step for the inlet and outlet boundary conditions.
+
+.. image:: images/boundary_conditions2.png
+    :align: center
+
+- To export the 2D mesh, select all the domains. In the **file** option, select **export** and **CAE**. Choose a name for the .msh file and press save. Your mesh should be done!
+
+.. warning::
+    The Deal.II mesh file parser cannot read the comments in the ``.msh`` file generated by Pointwise. Make sure to delete manually all the commented information in the file before launching a simulation.
+ 
+The boundary conditions identification numbers are chosen by Pointwise. The user has no control over the software's decision. The boundary condition IDs can be retrieved at the end of the ``.msh`` file generated. In the present case, the boundary IDs would correspond to 2 for the inlet, 3 for the outlet and 4 for all the walls.
+
+.. image:: images/mesh_file.png
+    :align: center
+    :width: 400px
+
+
+.. _moving_3d: 
+
+--------------------------
+Moving to 3D
+--------------------------
+
+This section will give you an idea of the tools that can be used to create the following 3D mesh.
+
+.. image:: images/final_mesh_3d2.png
+    :align: center
+
+----
+
+Extrusions
+-----------
+
+The best way to create 3D **block** meshes from 2D geometries is by using extrusions. The following steps can be achieved from the final 2D geometry generated previously. 
+
+- **Set Dimension** to 3D in the **CAE** menu
+- Select all the geometry and click on **Translate** in the **Edit** and **Transform** menu.
+- Insert **0.5 0 0** in the **Offset Vector** box and press enter. Click on **ok** to leave the menu. 
+
+.. image:: images/translation.png
+    :align: center
+
+The last action should have translated the geometry from 0.5 in the x direction. This translation is needed to avoid the creation of tetrahedral cells in the center of the mesh during the rotational extrusion. 
+
+- In the **Create** menu, select **Extrude** and **Rotate**
```

Comment:
Extrude and rotate have not been introduced before in the basics section? Is there a reason for it?

Reply:
Anybody familiar with CAD should be familiar with the extrusion concept.